plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/security/NugetApiKeyStoreImpl.java

```diff
@@ -62,6 +62,12 @@ public NugetApiKeyStoreImpl(final @Named("security") Provider<DatabaseInstance>
     this.databaseInstance = checkNotNull(databaseInstance);
     this.entityAdapter = checkNotNull(entityAdapter);
     this.principalsHelper = checkNotNull(principalsHelper);
+    try {
+      start();
+    }
+    catch (Exception e) {
+      e.printStackTrace();
```

Comment:
never use this in production code rethrow or log.

also not a good idea to start on ctor.